fullscreen.bs

```diff
@@ -503,11 +519,12 @@ properties apply to this pseudo-element either.
 <h3 id=:fullscreen-pseudo-class><code>:fullscreen</code> pseudo-class</h3>
 
 <p>The <dfn id=css-pc-fullscreen><code>:fullscreen</code></dfn> pseudo-class must match any
-<a>element</a> that has its <a>fullscreen flag</a> set.
-
-<p class="note no-backref">This makes it different from the {{Document/fullscreenElement}} API,
-which returns a <a>document</a>'s <a>fullscreen element</a>.
+<a>element</a> that has its <a>fullscreen flag</a> set, and the non-null results of
+<a>retargeting</a> <a>fullscreen element</a> against each <a>connected</a> <a for="tree">root</a> of
+<a>node trees</a>.
```

Comment:
This still reads weird. It needs to have both the flag set and be the result of retargeting?

Which fullscreen element are we talking about here? What does it belong to?

I'm also not sure that "root of node trees" is a thing or what that actually means.